www/docs/server/procedures.md

```diff
@@ -157,6 +157,27 @@ export const appRouter = router({
 export type AppRouter = typeof appRouter;
 ```
 
+### With [@effect/schema](https://github.com/Effect-TS/schema)
```

Comment:
Might be an idea to rebase onto my branch, as I've totally changed this doc, and actually I'm sceptical if we should be documenting anything which we don't want to commit to officially supporting? I've added a community section with just links to projects

Reply:
Let's use your branch

Reply:
valid take - but still good to know what validators we support and not?

Reply:
I've added to https://github.com/trpc/trpc/pull/4245/files and also enabled twoslash for it so we get some type-checking as tRPC evolves.

There may still be a case for the added test in this PR, I'm not sure though.

Reply:
yea there's definetely two sides of the coin here

Reply:
I'll pull in this file to my PR then, have a few other tests to add so might as well kill all the birds alongside the docs changes